tembo-cli/tembo/docker-compose.yml.template

```diff
@@ -17,27 +17,55 @@ services:
       - "traefik.tcp.routers.{{instance.instance_name}}.entrypoints=postgresql"
       - "traefik.tcp.routers.{{instance.instance_name}}.tls.passthrough=true"
       - "traefik.tcp.services.{{instance.instance_name}}.loadbalancer.server.port=5432"
+{% if instance.controller_app_services -%}
+{% for key, app_services in instance.controller_app_services -%}
 
-  {% if instance.app_services.restapi %}
-  {{instance.instance_name}}-postgrest:
-    image: postgrest/postgrest:v10.0.0
+{% set_global pathPrefixes = "" -%}
```

Comment:
The template is getting complex. I will look at simplifying this either by moving it to rust or figuring out another way in the coming PRs.

Reply:
I agree, at this point it seems like we should just have a Value type constructed in rust then formatted into the yaml file